workbench-core/example/infrastructure/src/__tests__/__snapshots__/example-stack.test.ts.snap

```diff
@@ -252,7 +252,7 @@ Object {
         },
         "FunctionVersion": Object {
           "Fn::GetAtt": Array [
-            "ExampleLambdaServiceCurrentVersionDB4EDA2B9b9c79fb44cad026e6114dcbb2adeec6",
```

Comment:
non-blocking: snapshot test has been deleted for example App in the latest merge to develop. So this file should not be present